Symfony/CS/Tests/FixerFactoryTest.php

```diff
@@ -279,4 +279,24 @@ function ($name) {
 
         return $cases;
     }
+
+    public function testHasRule()
```

Comment:
Not a must have, but it would be great to have it done on mocks instead of real fixers.
Now the fixer will fail if we rename the fixer name and forgot to change the test here. (second test too)
Any chances?